origin-mobile/android/app/build.gradle

```diff
@@ -148,6 +149,7 @@ android {
 }
 
 dependencies {
+    compile project(':react-native-sentry')
```

Comment:
Instead of `compile`, this should be `implementation`.  It's just a warning right now but not for long.

Reply:
Would you mind updating this if you think it is a good idea @mikeshultz? I have no idea what I'm doing with android builds :grin: